src/merkle/path.rs

```diff
@@ -106,3 +124,74 @@ pub struct RootPath {
     /// The path from `value` to `root` (exclusive).
     pub path: MerklePath,
 }
+
+#[cfg(test)]
+mod test {
+    use crate::merkle::{int_to_node, MerklePath, Rpo256, Vec, Word};
```

Comment:
We might want to add some edge cases such as when we have an empty path or single node to assert it won't panic + return a consistent result.

We also changed the implementation of `compute_root`, so it would be good to add some tests cases for that as well

Reply:
added a test for empty path, `compute_root` are implicitly test via `verify`